lib/ome/files/MetadataTools.cpp

```diff
@@ -548,7 +548,8 @@ namespace ome
           try
             {
               OMEXMLMetadata& omexml(dynamic_cast<OMEXMLMetadata&>(store));
-              addMetadataOnly(omexml, s);
+              if (omexml.getTiffDataCount(s) == 0)
```

Comment:
On `develop` the `BinData` count check has been added by @dgault recently in https://github.com/openmicroscopy/bioformats/commit/947c13a8bcf5b6ad195185bf8833b8d88e525776 but is there any other place that needs updating?